docs/content/specs/shared/interfaces.md

```diff
@@ -34,6 +34,12 @@ Allowed values for logs and metric categories or category groups **MUST NOT** be
   {{< /tab >}}
 {{< /tabs >}}
 
+{{< hint type=note >}}
+
+In the provided example for Diagnostic Settings, both logs and metrics are enabled for the associated resource. However, it is IMPORTANT to note that certain resources may not support both diagnostic setting categories. In such cases, the resource configuration MUST be modified accordingly to ensure proper functionality and compliance with system requirements.
```

Comment:
```suggestion
In the provided example for Diagnostic Settings, both logs and metrics are enabled for the associated resource. However, it is IMPORTANT to note that certain resources may not support both diagnostic setting types/categories. In such cases, the resource configuration **MUST** be modified accordingly to ensure proper functionality and compliance with system requirements.
```

Reply:
@jtracey93 - Included this and added emphasis to `IMPORTANT` as noticed this was missed